### ISSUE TITLE
Screen reader is not announcing body content of dialog in windows 11

### ISSUE THREAD

**Issue by agarwal-taniya** [microsoft/fluentui#19593]
### Environment Information

- **Package version(s)**: Fluent UI React 8.30.2
- **Browser and OS versions**: Chrome in windows 11
- Screen reader: JAWS 2021

### Describe the issue:

Fluent Element - Dialog

Screen reader does not announce sub text along with title within a dialog.

### Please provide a reproduction of the issue in a codepen:

See blocking dialog example on this page: https://developer.microsoft.com/en-us/fluentui#/controls/web/dialog

on a windows 11 machine

#### Actual behavior:

Screen reader does not announce sub text along with title within a dialog.

#### Expected behavior:

Screen reader should announce sub text along with title within a dialog.

**Comment by gouttierre** [comment]
@agarwal-taniya  - Thanks for filing this issue with us. To set expectation our team will have a look at the issue but only once capacity allows.
  
@bsunderhus -  Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley

**Comment by spmonahan** [comment]
Hi @agarwal-taniya. I've looked at this issue with @smhigley and we believe the issue is a bug in with JAWS as NVDA works correctly with the same content and Narrator works correctly in Edge. I've filed [a bug against JAWS](https://github.com/FreedomScientific/VFO-standards-support/issues/588) and will update this issue based on what happens with the JAWS bug.

**Comment by smhigley** [comment]
This doesn't reproduce for me with JAWS 2022 and Edge v101 on windows 11. The blocking dialog is an alertdialog (a form of live region). I'm guessing if there was an issue with the live region and JAWS in the past, it's been fixed now. Closing this issue since it no longer repros.
